SAMtags.tex

```diff
@@ -453,6 +455,106 @@ \subsubsection{Color space}
+strictly {\tt N}.  This is followed by either plus or minus
+indicating the strand the modification was observed on (relative to
+the recorded strand of {\sf SEQ} with plus meaning same
+orientation), and a base modification symbol.  This is then followed
+by a comma separated list of how many unmodified seq bases of the
+stated base type to skip, stored as a delta to the last and starting
+with 0 as the first (or next) base.  Hence this number series is
+comparable to the numbers in an {\tt MD} tag.
+
+For example {\tt C+m,5,12,0;} tells us there are three 5-Methylcytosine
+bases in the original {\sf SEQ}.  The first 5 {\tt C} bases are
+unmodified and the 6th is modified, as are the 19th (12 inbetween the
+6th and 19th) and 20th.  Similarly {\tt G-m,14;} indicates the 15th
+{\tt G} is a 5-Methylcytosine on the opposite strand.
+
+This permits modifications to be listed on either strand with the rare
+potential for both strands to have a modification at the same site.
+
+If the modification is not one of the standard common types (listed
+below) it can be specified as a numeric ChEBI code.  For example
+{\tt C+76792,57;} is the same as {\tt C+h,57;}.
+
+An unmodified base of {\tt N} means count any base in {\sf SEQ}, not
+only those of {\tt N}.  Thus {\tt N+n,100;} means the 101st base is
+Xanthosine (n), irrespective of the sequence composition.
+
+The standard code types and their associated ChEBI values are listed
+below, taken from \emph{Modeling methyl-sensitive transcription factor
+motifs with an expanded epigenetic alphabet}, Coby Viner
+et.al. \url{https://www.biorxiv.org/content/10.1101/043794v1}.
+Additionally ambiguity codes {\tt A}, {\tt C}, {\tt G} and {\tt T}
+exist to represent unspecified modifications bases of their respective
+canonical base types, plus code {\tt N} to represent an unspecified
+modification of any base type.
+
+\begin{center}
+\begin{tabular}{lllll}
+{\bf Unmodified base} & {\bf Code} & {\bf Abbreviation} & {\bf Name} & {\bf ChEBI} \\
+\hline
+C & m & 5mC   & 5-Methylcytosine        & 27551 \\
+C & h & 5hmC  & 5-Hydroxymethylcytosine & 76792 \\
+C & f & 5fC   & 5-Formylcytosine        & 76794 \\
+C & c & 5caC  & 5-Carboxylcytosine      & 76793 \\
+C & C &       & Ambiguity code; any C mod & \\
+\hline
+T & g & 5hmU  & 5-Hydroxymethyluracil   & 16964 \\
+T & e & 5fU   & 5-Formyluracil          & 80961 \\
+T & b & 5caU  & 5-Carboxyluracil        & 17477 \\
+T & T &       & Ambiguity code; any T mod & \\
+\hline
+A & a & 6mA   & 6-Methyladenine         & 28871 \\
+A & A &       & Ambiguity code; any A mod & \\
+\hline
+G & o & 8oxoG & 8-Oxoguanine            & 44605 \\
+G & G &       & Ambiguity code; any G mod & \\
+\hline
+N & n & Xao   & Xanthosine              & 18107 \\
+N & N &       & Ambiguity code; any mod & \\
+\end{tabular}
+\end{center}
+
+\item[MP:Z:\tagvalue{qualities}]
+\hfill\\
+The {\tt MP} tag if present lists the Phred qualities of each
```

Comment:
```suggestion
The (optional) {\tt MP} tag lists the Phred qualities of each
```

Reply:
Agreed, although I removed the parens around optional.